src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
```

Comment:
Ensure that the element with the class 'logo' exists in the DOM. If it doesn't, `document.querySelector('.logo')` will return null, and attempting to add an event listener to a null object will cause an error.